client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-category-list.tsx

```diff
@@ -125,12 +108,8 @@ const ScreenCategoryList = ( {
 							aria-describedby={ description }
 							aria-current={ isOpen }
 							onClick={ () => {
-								if ( isOpen ) {
-									setSelectedCategory( null );
-									onTogglePatternPanelList?.( false );
```

Comment:
I'd prefer to keep this feature to toggle the panel list. Let's wait for the final decision 🙂

Reply:
Okay. Let's keep it.